website/docs/sdks/index.md

```diff
@@ -49,55 +49,55 @@ If you see an item marked with a ❌ that you would find useful, feel free to re
 :::
 
 
-| Capability                                                                                        | [Java](/sdks/java_sdk) | [Node.js](/sdks/node_sdk) | [Go](/sdks/go_sdk) | [Python](/sdks/python_sdk) | [Ruby](/sdks/ruby_sdk) | [.NET](/sdks/dot_net_sdk) | [PHP](/sdks/php_sdk) | [Unleash Proxy Server](unleash-proxy.md) |
-|---------------------------------------------------------------------------------------------------|:----------------------:|:-------------------------:|:------------------:|:--------------------------:|:----------------------:|:-------------------------:|:--------------------:|:----------------------------------------:|
-| **Category: Initialization**                                                                      |                        |                           |                    |                            |                        |                           |                      |                                          |
-| Async initialization                                                                              | ✅                     | ✅                        | ✅                 | ✅                         | ✅                     | ✅                        | ✅                   | N/A                                      |
-| Can block until synchronized                                                                      | ✅                     | ✅                        | ⭕                 | ⭕                         | ⭕                     | ✅                        | ⭕                   | N/A                                      |
-| Default refresh interval                                                                          | 10s                    | 15s                       | 15s                | 15s                        | 15s                    | 30s                       | 30s                  | 5s                                       |
-| Default metrics interval                                                                          | 60s                    | 60s                       | 60s                | 60s                        | 60s                    | 60s                       | 30s                  | 30s                                      |
+| Capability                                                                                        | [Java](/sdks/java_sdk) | [Node.js](/sdks/node_sdk) | [Go](/sdks/go_sdk) | [Python](/sdks/python_sdk) | [Ruby](/sdks/ruby_sdk) | [.NET](/sdks/dot_net_sdk) | [PHP](/sdks/php_sdk) | [Unleash Proxy Server](unleash-proxy.md) | [Rust](/sdks/rust_sdk) |
```

Comment:
Here's why the build is failing, by the way: You can't use broken (internal) links 🤓 Suggest we keep linking to the github SDK for now:

```suggestion
| Capability                                                                                        | [Java](/sdks/java_sdk) | [Node.js](/sdks/node_sdk) | [Go](/sdks/go_sdk) | [Python](/sdks/python_sdk) | [Ruby](/sdks/ruby_sdk) | [.NET](/sdks/dot_net_sdk) | [PHP](/sdks/php_sdk) | [Unleash Proxy Server](unleash-proxy.md) | [Rust](https://github.com/unleash/unleash-client-rust) |
```

Reply:
Thank you!